engine/gameobject/src/gameobject/gameobject.cpp

```diff
@@ -237,6 +238,18 @@ namespace dmGameObject
         return regist->m_DefaultCollectionCapacity;
     }
 
+    void SetInputStackDefaultCapacity(HRegister regist, uint32_t capacity)
+    {
+        assert(regist != 0x0);
+        regist->m_DefaultInputStackCapacity = capacity;
+    }
+
+    uint32_t GetInputStackDefaultCapacity(HRegister regist)
```

Comment:
Shouldn't be needed outside of this cpp file?
I recommend adding "static" to this function.

Reply:
Yeah, you're right. The use site is in the same file. I can add a static.

Reply:
Done